### ISSUE TITLE
h2 protocol error: http2 error: connection error detected: frame with invalid size

### ISSUE THREAD

**Issue by iboss-ptk** [osmosis-labs/beaker#122]
The following error happens when the network endpoint requires TLS

```
Error: Account can't be initialized

Caused by:
    0: status: Unknown, message: "h2 protocol error: http2 error: connection error detected: frame with invalid size", details: [], metadata: MetadataMap { headers: {} }
    1: transport error
    2: http2 error: connection error detected: frame with invalid size
    3: connection error detected: frame with invalid size
```

suspect missing `tonic/tls` feature here:

https://github.com/cosmos/cosmos-rust/blob/3f7ec00098ac673ef8d982d7c363df535ac8dfa8/cosmos-sdk-proto/Cargo.toml#L24-L30

**Comment by dzmitry-lahoda** [comment]
same for me.

btw our centaurid daemon works ok with endpoint

```
name = ''
gas_price = '0.025ppica'
gas_adjustment = 1.3
account_prefix = 'centauri'

[networks.testnet]
chain_id = 'banksy-testnet-3'
network_variant = 'Shared'
#grpc_endpoint = 'https://grpc-t.composable.nodestake.top/443'
rpc_endpoint = 'https://rpc-t.composable.nodestake.top/443'
```

**Comment by iboss-ptk** [comment]
This should be fixed in `0.1.7`.
